### ISSUE TITLE
Has this incubation stalled?

### ISSUE THREAD

**Issue by marcoscaceres** [mikewest/lang-client-hint#7]
It appears there's been no progress on this incubation for a few years so it's looking like it stalled. 

Any objections to us archiving this repo? 

Unless we hear otherwise, we will archive it in a week or so.

**Comment by yoavweiss** [comment]
^^ @miketaylr

**Comment by miketaylr** [comment]
Yeah, I think it's fair to archive now @marcoscaceres @yoavweiss - we're exploring another solution (unclear if that will be what we ship), and if we decide to come back to something closer to a client hint, we can just un-archive.
